src/source-map.ts

```diff
@@ -18,7 +20,7 @@ export const installSourceMapSupport = () => {
 		 *
 		 * https://github.com/nodejs/node/blob/91193825551f9301b6ab52d96211b38889149892/lib/internal/errors.js#L141
 		 */
-		&& typeof Error.prepareStackTrace !== 'function'
+		&& isPrepareStackTraceExposed ? typeof Error.prepareStackTrace === 'function' : typeof Error.prepareStackTrace !== 'function'
```

Comment:
Confused what's happening here. It's checking if it's a function, and then if it's not a function.

What are we actually trying to check for here? Can you update the comment?